mllib/src/main/scala/org/apache/spark/mllib/clustering/BisectingKMeansModel.scala

```diff
@@ -33,7 +33,7 @@ import org.apache.spark.rdd.RDD
 @Since("1.6.0")
 @Experimental
 class BisectingKMeansModel @Since("1.6.0") (
```

Comment:
Just realized this constructor should be private since the argument is private.  As long as you're updating this, could you please eliminate the now-unnecessary Since annotations here?